addons/guide_10_app_exclusions/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+    "api_version": "0.1",
+    "id": "guide_10_app_exclusions",
+    "name": "Guide: app exclusions",
```

Comment:
Would it make sense to somehow nest this into the same folder as app permissions, since it's essentially the same guide for a different version of the app?

Reply:
2 manifests in the same folder? This is not supported.

Reply:
yea essentially. I was thinking more along the lines of

guide_07_app_permissions
↳ V2.13
  ↳ manifest.json
↳ V2.14
  ↳ manifest.json

Reply:
nice but we need to change the addon script to make it happen. Can we do it in a follow up?